Gemfile

```diff
@@ -40,13 +42,13 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'rspec-rails'
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
```

Comment:
Just curious if you have any idea what is going on here?

Reply:
This resolves a warning from bundler. Before, if you ran bundler you would get a huge yellow warning saying it was only installing certain dependencies for ruby. Now if you run it, there will be no warnings :)

Reply:
Is this notation only important for bundler or should we be using in other locations. I believe it was also changed in routes.rb?